### ISSUE TITLE
Support prefix rewrites for zed import just like the playground does

### ISSUE THREAD

**Issue by glorious-beard** [authzed/zed#81]
When one edits a schema in the playground then imports it back into a permissions system via the "import" button, the playground automatically prepends the permission system's prefix to each reference in the schema.

However, that rewrite doesn't happen on `zed import` operations, making it difficult to share a common schema file with multiple permission systems; each reference in the schema has to be manually modified to include the prefix for the destination permission system.

This is a request to either:
- fetch the prefix from the destination permissions system and then automatically prepend the prefix to each reference, just like the playground (ideal)
- pass in the prefix via a command line option (not as ideal)

**Comment by glorious-beard** [comment]
Discord thread link:
https://discord.com/channels/844600078504951838/844600078948630559/935981273552404540

**Comment by jzelinskie** [comment]
I've actually raised this in the past, so I'm glad to see that my failure to create an issue was accounted for, haha.
We've discussed having a button to help rewrite prefixes in the playground, which is another "non-ideal" way to handle the prefixes.

One downside with looking up the the prefix for the given credential is that if there is ever a future where Authzed.com users can share/bridge their schemas with each other, this could lead to some very subtle bugs.
